code-spot-client/src/app/code-editor/code-editor.component.html

```diff
@@ -8,7 +8,7 @@
     <div class="spinner-grow text-primary mb-3" role="status"></div>
     <br />
     <h2>Loading...</h2>
-    <p>This will only take a few seconds.</p>
+    <p>We're preparing an advanced coding environment for you.</p>
```

Comment:
Tao thấy từ advanced hơi mạnh :) Dùng từ collaborative đc ko?
"We're preparing a **collaborative** coding environment for you."